README.md

```diff
@@ -11,10 +11,17 @@ Decode: python3 tfci.py decompress ../data/image_compression_benchmark/big_build
 # K-means clustering-based codec
 ## Run codec
 python3 kmeans.py [path of image to be compressed] compressed/[name of compressed image file] reconstructed/[name of decoded image file] [number of clusters]
+<!-- edit2 -->
+
 
 # Classical algorithms - JPEG/PNG
 ## Build codec
+<!-- edit 4 -->
```

Comment:
+
 
### Decode:
### Reconstruct:

---

Please provide a review for the diff.

I have made some changes in the README.md file of the given repository on GitHub, as shown above. I would like to know if my changes are good or not. Please provide your valuable feedback so that I can improve the file further.